cmd/podman/images/list.go

```diff
@@ -184,16 +184,13 @@ func writeTemplate(imageS []*entities.ImageSummary) error {
 }
 
 func tokenRepoTag(tag string) (string, string) {
-	tokens := strings.SplitN(tag, ":", 2)
-	switch len(tokens) {
-	case 0:
+	tagSplitIndex := strings.LastIndex(tag, ":")
+	if tagSplitIndex == -1 {
 		return tag, ""
-	case 1:
-		return tokens[0], ""
-	case 2:
-		return tokens[0], tokens[1]
-	default:
-		return "<N/A>", ""
```

Comment:
Hm. I feel like we might be missing this, but I'm not 100% sure how it's determined